src/components/LibraryItem.tsx

```diff
@@ -126,6 +109,23 @@ export class LibraryItem extends React.Component<LibraryItemProps, LibraryItemSt
             nestedElements = this.getNestedElements(groupedItems);
         }
 
+        if (this.props.indentLevel > 0) {
+            let indents = [];
+            let indentationIconRootPath = "/src/resources/ui/";
+
+            for (let i = 1; i <= this.props.indentLevel; i++) {
+                let indentationPath = indentationIconRootPath;
+                if (i == this.props.indentLevel) {
+                    indentationPath += clusteredElements ? "indent-line-l.svg" : "indent-line-t.svg";
+                } else {
+                    indentationPath += "indent-line-i.svg";
+                }
+                indents.push(<img key={i} className={"Indentation"} src={indentationPath} />);
+            }
+
+            indentation = (<div className={"Indents"}>{indents}</div>);
+        }
+
```

Comment:
Can you help us reduce the above lines to:

```js
let indentElements = this.getIndentElements();
```

The `render()` function gets a little too long now.